Bindings/bindings.json

```diff
@@ -474,6 +474,124 @@
             ]
         },
         {
+            "type": "apiHubTable",
+            "displayName": "SaaS Table",
```

Comment:
I didn't even realize there was a variables syntax.  It looks like we're not using it very consistently, but I'll add it for this binding type.